components/admin/requests/Header.tsx

```diff
@@ -107,28 +117,62 @@ export default function RequestHeader({
     return null;
   };
 
+  const displayShopifyUrl = paidThroughShopify && shopifyOrderID && shopifyOrderNumber;
+
+  const shopifyOrderUrl = displayShopifyUrl
+    ? `https://${process.env.NEXT_PUBLIC_SHOPIFY_DOMAIN}/admin/orders/${shopifyOrderID}`
+    : undefined;
```

Comment:
```suggestion
    : null;
```
Semantically, this should be `null`